tests/e2e/columns-tasks.spec.js

```diff
@@ -0,0 +1,60 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+const TEST_LINKS = ['/act', '/explore', '/'];
+
+test('Test Columns block with Tasks style', async ({page, context}) => {
+  // Login.
+  await page.goto('./');
```

Comment:
Can we drop this step(go to homepage)?   🤔

Reply:
I don't think so 🤔 Otherwise the browser opens but doesn't go to the site... Maybe we could make it part of the `login` function though, so that we don't have to write it down each time